modules/ROOT/pages/websockets/websockets-connector-messaging-operations.adoc

```diff
@@ -4,11 +4,33 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: connectors, ws, wss, websocket, broadcast
 
-== About WebSockets Broadcasting
+== About WebSockets Operations
+
+=== Send Operation
+
+The send operation allows the user to send a message through WebSocket identified by its socketId, once a connection has been established. This operation can be used from both ends of the connection, by Inbound and Outbound WebSockets.
```

Comment:
can be used from both ends of the connection, by Inbound and Outbound WebSockets. -> can be used for both  Inbound and Outbound WebSockets.